pkg/server/server.go

```diff
@@ -60,35 +64,42 @@ func NewKrokServer(cfg Config, deps Dependencies) *KrokServer {
 
 // Run starts up listening.
 func (s *KrokServer) Run(ctx context.Context) error {
-	// Setup Global Token Key
-	if s.Config.GlobalTokenKey == "" {
-		s.Logger.Info().Msg("Please set a global secret key... Randomly generating one for now...")
-		b := make([]byte, 32)
-		_, err := rand.Read(b)
-		if err != nil {
-			return err
-		}
-		state := base64.StdEncoding.EncodeToString(b)
-		s.Config.GlobalTokenKey = state
-	}
 	s.Dependencies.Logger.Info().Msg("Start listening...")
+
 	// Echo instance
 	e := echo.New()
 
 	// Middleware
 	e.Use(middleware.Logger())
 	e.Use(middleware.Recover())
+	e.Use(middleware.CORSWithConfig(middleware.CORSConfig{
+		AllowOrigins:     []string{"http://localhost:3000"},
```

Comment:
Will need to update this dynamically. When using credentials/cookies you can't just `*` this unfortunately!